GOVERNANCE.md

```diff
@@ -131,6 +132,15 @@ RFCs are not regarded as documentation. The official specification and implement
 ## Roadmap
 The Cloud Native Buildpacks TOC is responsible for establishing a yearly roadmap laying out our aspirations for that year. This shared vision is essential for keeping the development process focused.
 
+## Vendor Neutrality
+The Cloud Native Buildpacks TOC is committed to ensuring the vendor neutrality of the project in accordance with the [CNCF guidelines on what it means for a project to be vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+
+No company may hold a majority (fifty percent or more) of the total TOC seats through any combination of seats. If the results of an election result in greater than two employees of the same organization, the lowest vote getters from any particular employer will be removed until representation on the committee is down to two from that organization.
+
+If employers change because of job changes, acquisitions, or other events, in a way that would violate the company representation rules above, sufficient members of the committee must resign until the composition rules are met. If it is impossible to find sufficient members to resign, all employees of that organization will be removed and new special elections held. In the event of a question of company membership (for example evaluating independence of corporate subsidiaries) a majority of all non-involved TOC members will decide.
```

Comment:
it could also happen that the toc is only left with a single company representation as a result of company changes? what happens then?

Reply:
We could add a provision when that happens.

Reply:
Yea, i think our goal here is to show good intent, versus covering all possible scenarios. If only one vendor is participating in the project there's only so much we can do... other than show good intent. I say we add a provision if that happens.